docker-compose.yml

```
@@ -6,13 +6,17 @@ services:
       context: .
       dockerfile: docker/Dockerfile
       args:
-        - PHP_VERSION=${PHP_VERSION:-7.1}
+        - PHP_VERSION=${PHP_VERSION:-7.2}
         - XDEBUG=${XDEBUG:-0}
     volumes:
       - ./:/docker
       - ./docker/php.ini:/usr/local/etc/php/php.ini
     depends_on:
       - mongo
-
+    environment:
+        TRAVIS: ${TRAVIS}
+        TRAVIS_JOB_ID: ${TRAVIS_JOB_ID}
+        TRAVIS_BRANCH: ${TRAVIS_BRANCH}
+        TRAVIS_PULL_REQUEST: ${TRAVIS_PULL_REQUEST}
```

This more concise syntax should have the same effect:

```suggestion
        - TRAVIS
        - TRAVIS_JOB_ID
        - TRAVIS_BRANCH
        - TRAVIS_PULL_REQUEST
```